app/routes/questions/$question.tsx

```diff
@@ -173,7 +173,7 @@ export function Question({
 const updateTextNodes = (el: Node, textProcessor: (node: Node) => Node) => {
   Array.from(el.childNodes).forEach((child) => updateTextNodes(child, textProcessor))
 
-  if (el.nodeType == Node.TEXT_NODE && el.textContent && el?.parentElement?.tagName != 'A') {
```

Comment:
please check git commit for the old code, it was fixing (workarounding?) some previous bug - not sure which, but it deserves to be re-tested that removing the last condition won't reintroduce the previous bug 🤞

Reply:
it was checking whether the glossary item is part of a link, so it didn't display links as glossary items. But apparently the editors want that... :D